src/ksm/group.go

```diff
@@ -22,7 +22,7 @@ type ksmGrouper struct {
 // addServiceSpecSelectorToGroup adds a new metric to the service group
 // which includes the selectors defined in the service spec.
 func (r *ksmGrouper) addServiceSpecSelectorToGroup(serviceGroup map[string]definition.RawMetrics) error {
-	services, err := r.k8sClient.ListServices()
+	services, err := r.k8sClient.ListServices("")
```

Comment:
This line makes me wonder if we have any kind of caching set up for calls like this.

Reply:
I don't think so, but it would be easier if we could implement that once we have a daemon process, so we can cache in memory. Using e.g. informers.